eo-parser/pom.xml

```diff
@@ -33,6 +33,10 @@ SOFTWARE.
   <packaging>jar</packaging>
   <name>eo-parser</name>
   <description>EO-to-XML Parser</description>
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
```

Comment:
@nlchar no need, we have this in the parent: https://github.com/jcabi/jcabi-parent/blob/master/pom.xml#L104-L105